config.yaml

```diff
@@ -61,12 +61,10 @@ options:
     type: string
     description: | 
       IP address of SAN controller for SSH access to the array
-    default: ''
```

Comment:
I think all of the default values for the config.yaml should just be something like (note just the blank default):

```yaml
san-ip:
  type: string
  default:
  description: |
    IP address of SAN controller for SSH access to the array
```